presto-main/src/test/java/com/facebook/presto/execution/TestInput.java

```diff
@@ -32,7 +32,8 @@ public void testRoundTrip()
         Input expected = new Input(new ConnectorId("connectorId"), "schema", "table", Optional.empty(), ImmutableList.of(
                 new Column("column1", "string"),
                 new Column("column2", "string"),
-                new Column("column3", "string")));
+                new Column("column3", "string")),
+                Optional.empty());
```

Comment:
Add a separate test that checks that TableStatistics are serializable.

Reply:
Done